Joplin_install_and_update.sh

```diff
@@ -71,7 +71,7 @@ if [[ ! -e ~/.joplin/VERSION ]] || [[ $(< ~/.joplin/VERSION) != "$version" ]]; t
 
     # Create icon for Gnome
     echo 'Create Desktop icon.'
-    if [[ $desktop =~ .*gnome.*|.*kde.*|.*xfce.*|.*mate.*|.*lxqt.*|.*unity.*|.*x-cinnamon.* ]]
+    if [[ $desktop =~ .*gnome.*|.*kde.*|.*xfce.*|.*mate.*|.*lxqt.*|.*unity.*|.*x-cinnamon.*|.*deepin*. ]]
```

Comment:
shouldnt it be 
`|.*deepin.*`
?

Reply:
Yeah, work too.
Thanks

Reply:
yes in the present use case it works too ;) 

I put that for the fun ;)

If i'm not wrong

`.*deepin*.` means any car 0 or more time  followed by deepi then followed by  the letter "n" 0 or more time, followed by any other car optionnaly

so in that case "deepik" will work too like deepinnnnnnn and deepinnnnnnz but not deepinnnnnnzz

and 
`.*deepin.*` means any car 0 or more time followed by deepin then followed by any car 0 or more time 

here foodeepinbar should be ok

Reply:
Interesting ! Thanks ;)